### PR TITLE
Require a strong password

### PR DESCRIPTION
**Pull request by StefanvdBroek**

### CODE REVIEW

File: src/CmsBundle/Entity/User.php [hunk 1]
```diff
@@ -56,8 +56,20 @@ class User extends FOSUser implements TwoFactorInterface
...
-     * @var string
+     * @Assert\Length(
+     *      min = 8,
+     *      max = 50,
+     *      minMessage = "Your password must be at least {{ limit }} characters long",
+     *      maxMessage = "Your password cannot be longer than {{ limit }} characters"
+     * )
+     *
+     * @Assert\Regex(
+     *     pattern="/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]+$/",
```
Comment (rvanlaarhoven) [inline src/CmsBundle/Entity/User.php]: This does not match `Afgjhdf5kgklfBhgdkhkh}df`

File: src/CmsBundle/Entity/User.php [hunk 2]
```diff
@@ -56,8 +56,20 @@ class User extends FOSUser implements TwoFactorInterface
     protected $email;
 
     /**
-     * @var string
+     * @Assert\Length(
+     *      min = 8,
+     *      max = 50,
```
Comment (rvanlaarhoven) [inline src/CmsBundle/Entity/User.php]: Why is 50 characters the max? I'd rather say; the max is the maximum amount of characters we allow in the database, which is 255.

**Review by rvanlaarhoven — CHANGES_REQUESTED** [review]

File: src/CmsBundle/Entity/User.php (hunk 1 above)
Comment (StefanvdBroek) [reply src/CmsBundle/Entity/User.php]: fixed

File: src/CmsBundle/Entity/User.php (hunk 2 above)
Comment (StefanvdBroek) [reply src/CmsBundle/Entity/User.php]: Changed to 255

**Review by rvanlaarhoven — APPROVED** [review]

**Pull request merged**